pkg/apis/core/types.go

```diff
@@ -5494,6 +5506,15 @@ type PodLogOptions struct {
 	// the actual log data coming from the real kubelet).
 	// +optional
 	InsecureSkipTLSVerifyBackend bool
+
+	// Return the specific log stream. Defaults to "All".
+	// Acceptable values are:
+	// - "All": the combined stdout and stderr
+	// - "stdout": only the stdout
+	// - "stderr": only the stderr
+	// This is an alpha field and requires enabling SplitStdoutAndStderr feature gate.
+	// +optional
+	Stream *LogStreamType
```

Comment:
One thing I remember we discussed in the past is how this option will work with the `LimitBytes` and `TailLines`. Ideally add a test that those log options will be respected alongside with setting the stream option